src/Common/Core/Impl/Services/ServiceBag.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Common.Core.Services {
+    public sealed class ServiceBag: IServiceBag {
```

Comment:
Basically in the next change I want to add ICoreServices that will hold IProcessServices, IFileSystem, IRegistry, IActionLog and ICoreShell (and more wrappers we care to add) so we can pass around fewer parameters. It is exported as ICoreServices.